src/common/settingsProvider.ts

```diff
@@ -5,10 +5,15 @@ import { SETTINGS_STORE_NAME } from '../utils';
 import { ThemeString } from './webviewEvents';
 
 const SUPPORTED_THEMES = new Map<string, string>([
-  ['Default Light+', 'light'],
-  ['Visual Studio Light', 'light'],
+  ['Default Light+', 'default'],
+  ['Visual Studio Light', 'default'],
```

Comment:
I don't think this code is required to fix this bug.

Reply:
The DevTools actually use "default" instead of "light" for the light theme, but since it defaults to the light theme when it doesn't know the string it makes no visible difference. Since I was changing the theme strings anyway, I thought I'd "fix" this, but it's technically not needed.